server/views/components/sendyform/subscribeform_sendy.html

```diff
@@ -0,0 +1,898 @@
+			font-family:Helvetica;
+			font-size:14px;
+			color:#333333;
+			padding:20px;
+		}
+		a:link,a:active,a:visited,a{
+			color:#336699;
+		}
+		.formEmailButton:link,.formEmailButton:active,.formEmailButton:visited,.formEmailButton,.formEmailButton span{
+			background-color:#5d5d5d !important;
+			color:#ffffff !important;
+		}
+		.formEmailButton:hover{
+			background-color:#444444 !important;
+			color:#ffffff !important;
+		}
+		label{
+			line-height:150%;
+			font-family:Helvetica;
+			font-size:16px;
+			color:#5d5d5d;
+		}
+		.field-group input,select,textarea,.dijitInputField{
+			font-family:Helvetica;
+			color:#5d5d5d !important;
+		}
+		.asterisk{
+			color:#cc6600;
+			font-size:20px;
+		}
+		label .asterisk{
+			visibility:hidden;
+		}
+		.indicates-required{
+			display:none;
+		}
+		.field-help{
+			color:#777;
+		}
+		.error,.errorText{
+			color:#e85c41;
+			font-weight:bold;
+		}
+	@media (max-width: 620px){
+		body{
+			width:100%;
+			-webkit-font-smoothing:antialiased;
+			padding:10px 0 0 0 !important;
+			min-width:300px !important;
+		}
+
+}	@media (max-width: 620px){
+		.wrapper,.poweredWrapper{
+			width:auto !important;
+			max-width:600px !important;
+			padding:0 10px;
+		}
+
+}	@media (max-width: 620px){
+		#templateContainer,#templateBody,#templateContainer table{
+			width:100% !important;
+			-moz-box-sizing:border-box;
+			-webkit-box-sizing:border-box;
+			box-sizing:border-box;
+		}
+
+}	@media (max-width: 620px){
+		.addressfield span{
+			width:auto;
+			float:none;
+			padding-right:0;
+		}
+
+}	@media (max-width: 620px){
+		.captcha{
+			width:auto;
+			float:none;
+		}
+
+}		.gdpr-mergeRow{
+			margin:10px 0;
+			color:#4a4a4a;
+			font-family:Helvetica;
+		}
+		.gdpr-content{
+			margin:0 -20px 0 -20px;
+			padding:20px;
+			background:rgba(255, 255, 255, 0.8);
+		}
+		#gdpr-label{
+			margin-top:0;
+		}
+		#gdpr-description{
+			margin-top:6px;
+			margin-bottom:0;
+			padding-bottom:11px;
+		}
+		.gdpr-content p{
+			color:#4a4a4a;
+			font-size:13px;
+			line-height:1.5;
+		}
+		.gdpr-content .checkbox-group label{
+			font-weight:normal;
+			font-size:13px;
+		}
+		#gdpr-legal{
+			padding-top:20px;
+			margin:0;
+		}
+		.gdpr-footer{
+			margin:0 -20px 0 -20px;
+			padding:20px;
+			background:rgba(255, 255, 255, 0.9);
+			overflow:auto;
+			color:#6d6d6d;
+		}
+		.gdpr-footer p{
+			font-size:10px;
+			line-height:1.5;
+			margin-bottom:0;
+		}
+		.gdpr-footer a{
+			color:#206578;
+		}
+		.gdpr-footer img{
+			width:65px;
+			float:left;
+			margin-right:15px;
+		}
+</style></head>
+
+
+	<body>
+		<div class="wrapper rounded6" id="templateContainer">
+		    <h1 class="masthead"><img src="https://github.com/scbadkas19/website/blob/mailchimptosendyform/server/views/components/sendyform/BIsendyformimage.png" alt="" border="0" style="border:0px  ; border-color:; border-style:; border-width: 0px; height: 500px; width: 1500px; margin: 0; padding: 0;" width="1500" height="500"></h1>

```

Comment:
Either change all absolute links to `blockchainedindia` repo links or change to relative urls or call from assets static resources.